packages/create-plugin/templates/github/workflows/ci.yml

```diff
@@ -203,6 +206,14 @@ jobs:
         id: run-tests
         run: {{ packageManagerName }} run e2e
 
+      - name: Upload e2e test summary
+        uses: grafana/plugin-actions/playwright-gh-pages/upload-report-artifacts@main
+        if: $\{{ always() && !cancelled() }}
+        with:
+          upload-report: false
```

Comment:
This means the report won't be uploaded to GH artifacts, and therefore not deployed to GH pages in the next job.